test/marketo/write/write.go

```diff
@@ -17,15 +17,22 @@ func main() {
 }
 
 func MainFn() int {
-	err := testWrite(context.Background())
+	ctx := context.Background()
```

Comment:
After you update the tests for both kinds of objects, please add screenshots of it in the PR